app/views/endemics/declaration.njk

```diff
@@ -15,7 +15,7 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">Review your agreement offer</h1>
 
-    <p class="govuk-body">This is an offer of funding for a {{ species }} review for this business:</p>
+    <p class="govuk-body">This is an offer of funding for a {{ species }} review and follow-up for this business:</p>
```

Comment:
Doesn't match these designs: https://fcp-ahwr-prototype.herokuapp.com/v23/apply/declaration

Reply:
If you mean "livestock health and welfare" is not added, that is because I think {{species}} should return the "livestock health and welfare", am I right?